pkg/agent/ocp.go

```diff
@@ -47,12 +63,66 @@ func NewClusterOpenShiftAPIClient(ctx context.Context, assetDir string) (*Cluste
 
 }
 
+// AreClusterOperatorsInitialized Waits for all Openshift cluster operators to initialize
+func (ocp *ClusterOpenShiftAPIClient) AreClusterOperatorsInitialized() (bool, error) {
+
+	failing := configv1.ClusterStatusConditionType("Failing")
+
+	operators, err := ocp.ConfigClient.ConfigV1().ClusterOperators().List(ocp.ctx, metav1.ListOptions{})
```

Comment:
This should be `ClusterVersions()`

This is causing the check on line 79 to always fail, because `clusteroperator` does not have a condition named 'Failing`, only `clusterversion` does.